Source/KSCrash/Recording/Monitors/KSCrashMonitor_AppState.c

```diff
@@ -369,24 +385,35 @@ bool kscrashstate_reset()
         g_state.launchesSinceLastCrash++;
         g_state.sessionsSinceLastCrash++;
         g_state.applicationIsInForeground = true;
-        
+
         return saveState(g_stateFilePath);
     }
     return false;
 }
 
+void kscrashstate_notifyObjCLoad(void)
+{
+    KSLOG_TRACE("KSCrash has been loaded!");
+    g_state.applicationIsInForeground = false;
```

Comment:
May we call `memset` here and not setting values directly in `kscrashstate_initialize`?

Reply:
Yes, we may